utils.py

```diff
@@ -138,3 +138,11 @@ async def helper_plus(ctx):
         if role.id in allowed_roles:
             return True
     raise NotHelperPlusError
+
+
+def add_author_footer(embed: discord.Embed, ctx: discord.ext.commands.Context,
+                      set_timestamp=True, additional_text: str = ""):
+    if set_timestamp:
+        embed.timestamp = datetime.now(tz=timezone.utc)
+
+    embed.set_footer(icon_url=ctx.author.avatar_url, text=str(ctx.author)+additional_text)
```

Comment:
```suggestion
    embed.set_footer(icon_url=ctx.author.avatar_url, text=f"{ctx.author} {additional_text}")
```
This looks a bit better IMO. Also note moving space to string instead of passing it as first character of parameter. This is not blocking comment.